client/app/lib/providers/templates/google.coffee

```diff
@@ -46,7 +46,7 @@ resource:
     # this is the name of your VM
     google-instance:
       # and this is its identifier (required)
-      name: google-instance
+      name: 'koding-${var.koding_group_slug}-${var.koding_stack_id}-${count.index+1}'
```

Comment:
Could you remove https://github.com/koding/koding/blob/c8c894efcb80d89cd4d49f43ef737a1bc148c289/go/src/koding/kites/kloud/provider/google/stack.go#L201 from google as well?